assets/javascripts/discourse/components/ai-persona-tool-options.gjs

```diff
@@ -0,0 +1,81 @@
+import Component from "@glimmer/component";
+import I18n from "discourse-i18n";
+import AiPersonaToolOptionEditor from "./ai-persona-tool-option-editor";
+
+export default class AiPersonaToolOptions extends Component {
+  get showToolOptions() {
+    const allTools = this.args.allTools;
```

Comment:
I don't think there is any need to store this in a separate const, you don't do it in `toolNames`

Reply:
its so minor but it avoids needing to do this.args.allTools twice, certainly not a hill I would die on.